pkg/cmd/edge_applications/init/init.go

```diff
@@ -419,17 +400,12 @@ func yesNoFlagToResponse(info *InitInfo) bool {
 	return false
 }
 
-func InitJavascript(info *InitInfo, cmd *InitCmd, conf *contracts.AzionApplicationConfig, envs []string) error {
+func InitStatic(info *InitInfo, cmd *InitCmd, conf *contracts.AzionApplicationConfig, envs []string) error {
 	pathWorker := info.PathWorkingDir + "/worker"
```

Comment:
Two questions: - 1. Is this used or are we using the initStatic below? 2. for Static type, is this worker directory still relevant??

Reply:
no used